### ISSUE TITLE
[Feat]: dark mode on login page

### ISSUE THREAD

**Issue by niwla23** [coder/code-server#5634]
## What is your suggestion?
adding a dark mode to the login page. It should use the user preference but it would be nice if you could also force it through a setting.

## Why do you want this feature?
dark mode good

## Are there any workarounds to get this functionality today?
user styles, dark reader

## Are you interested in submitting a PR for this?
yes after #5633  is merged so i don't have to rebase :)

**Comment by code-asher** [comment]
Closing in favor of https://github.com/coder/code-server/issues/3019

**Comment by sickplanet** [comment]
@niwla23 
try this https://github.com/sickplanet/code-server-dark-mode-login-page
